packages/taxdat/R/setup_helpers.R

```diff
@@ -454,6 +454,7 @@ check_set_tfrac <- function(set_tfrac) {
 #'
 check_snap_tol <- function(snap_tol, res_time) {
     if (!is.null(snap_tol)) {
+      snap_tol <- eval(parse(text = snap_tol))
```

Comment:
This line will fail if snap_tol cannot be evaluated. Can you add an error with a useful message if it fails?

Reply:
added error message if it fails using tryCatch function